src/idl_gen_swift.cpp

```diff
@@ -902,8 +899,8 @@ class SwiftGenerator : public BaseGenerator {
             GenerateStructArgs(*field.value.type.struct_def, &code, "", "",
                                "$0", true);
             code = code.substr(0, code.size() - 2);
-            code_ += "let __" + name + " = obj." + name + ".map { create" +
-                     field.value.type.struct_def->name + "(" + code + ") }";
+            code_ += "let __" + name + " = obj." + name + ".map { " + NameWrappedInNameSpace(*field.value.type.struct_def) + ".create" +
+                     Name(*field.value.type.struct_def) + "(" + code + ") }";
           } else {
```

Comment:
Fyi, if the tests fails, it would be because the cpp files aren't formatted to match the `clang-format` google standards.